e2e/benchmarks/local-benchmark/index.html

```diff
@@ -178,6 +179,8 @@ <h2>TensorFlow.js Model Benchmark</h2>
 
         // load model and run inference
         try {
+          // TODO: Remove tf.ready() when resolve issue on WebGPU backend.
```

Comment:
LGTM. nit: Do we have any details about the issue? Otherwise we can just remove this TODO?

Reply:
Please put `await tf.ready();` before ` await tf.setBackend` and comment that tf.ready() is needed for webgpu.